### ISSUE TITLE
panic: duplicate metrics collector registration attempted

### ISSUE THREAD

**Issue by deep2chain** [orientwalt/htdf#91]
panic when starting with prometheus = true in config.toml
```
$ hsd start
>>>
I[2021-08-31|00:13:10.989] Starting ABCI with Tendermint                module=main 
config.Prometheus true
config.PrometheusListenAddr :26660
INFO[0000] /---------flag1----------/ true              
config.Prometheus true
config.PrometheusListenAddr :26660
panic: duplicate metrics collector registration attempted

goroutine 1 [running]:
github.com/prometheus/client_golang/prometheus.(*Registry).MustRegister(0xc0002a7630, 0xc000633fe0, 0x1, 0x1)
        /media/gustav/OS/go/pkg/mod/github.com/prometheus/client_golang@v1.8.0/prometheus/registry.go:401 +0xb7
github.com/prometheus/client_golang/prometheus.MustRegister(...)
        /media/gustav/OS/go/pkg/mod/github.com/prometheus/client_golang@v1.8.0/prometheus/registry.go:178
github.com/go-kit/kit/metrics/prometheus.NewCounterFrom(0x1417fac, 0x4, 0x14171e2, 0x2, 0x14268b9, 0x11, 0x14268a8, 0x11, 0x0, 0xc000633fa0, ...)
        /media/gustav/OS/go/pkg/mod/github.com/go-kit/kit@v0.10.0/metrics/prometheus/prometheus.go:24 +0xf7
github.com/orientwalt/htdf/app/v0.PrometheusMetrics(0xc001014d20, 0xc000551bd8)
        /media/gustav/Investigation/github.com/orientwalt/htdf/app/v0/metrics.go:23 +0x152
github.com/orientwalt/htdf/app/v0.NewProtocolV0(0x0, 0x1719090, 0xc00118ce00, 0x170a728, 0xc000556fc0, 0xc0000ef570, 0x0, 0xc001014d20, 0xc000f6f800)
        /media/gustav/Investigation/github.com/orientwalt/htdf/app/v0/protocol_v0.go:113 +0x20f
github.com/orientwalt/htdf/app.NewHtdfServiceApp(0x1719090, 0xc00118ce00, 0xc0010e5530, 0x1728370, 0xc000550110, 0x0, 0x0, 0x1, 0x0, 0xc00161f808, ...)
        /media/gustav/Investigation/github.com/orientwalt/htdf/app/app.go:121 +0x75d
main.newApp(0x1719090, 0xc00118ce00, 0x1728370, 0xc000550110, 0x0, 0x0, 0xc0010e5530, 0xc000f998c0, 0xb24a6c)
        /media/gustav/Investigation/github.com/orientwalt/htdf/cmd/hsd/main.go:103 +0x19b
github.com/orientwalt/htdf/server.startInProcess(0xc00029aab0, 0x15abbe0, 0x0, 0x0)
        /media/gustav/Investigation/github.com/orientwalt/htdf/server/start.go:93 +0x176
github.com/orientwalt/htdf/server.StartCmd.func1(0xc00103e840, 0x2116448, 0x0, 0x0, 0x0, 0x0)
        /media/gustav/Investigation/github.com/orientwalt/htdf/server/start.go:54 +0xa5
github.com/spf13/cobra.(*Command).execute(0xc00103e840, 0x2116448, 0x0, 0x0, 0xc00103e840, 0x2116448)
        /media/gustav/OS/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:850 +0x472
github.com/spf13/cobra.(*Command).ExecuteC(0xc0001a3340, 0xc00103df40, 0x144362b, 0xc000f99ef0)
        /media/gustav/OS/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:958 +0x375
github.com/spf13/cobra.(*Command).Execute(...)
        /media/gustav/OS/go/pkg/mod/github.com/spf13/cobra@v1.1.1/command.go:895
github.com/tendermint/tendermint/libs/cli.Executor.Execute(0xc0001a3340, 0x15ac020, 0x141dd4e, 0x9)
        /media/gustav/OS/go/pkg/mod/github.com/orientwalt/tendermint@v0.99.5/libs/cli/setup.go:89 +0x3c
main.main()
        /media/gustav/Investigation/github.com/orientwalt/htdf/cmd/hsd/main.go:81 +0x797
```

**Comment by deep2chain** [comment]
https://github.com/prometheus/client_golang/issues/716

**Comment by deep2chain** [comment]
resolved at commit: 9674f1633a39566696c5b8eb6b1acd01487a86b5
